### PR TITLE
W: [fr] Command Line Options

### PR DESCRIPTION
**Pull request by trebmuh**
Supersedes #80 and partially based on it.

@ann0see : ready to be merged from my point of view.

### CODE REVIEW

File: wiki/fr/fr-Command-Line-Options.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+Si les fonctions les plus courantes de Jamulus peuvent être configurées à l'aide de l'interface graphique, Jamulus peut également être contrôlé à l'aide de ces options lorsqu'il est démarré en ligne de commande. La manière exacte de procéder dépendra de votre système d'exploitation.
+
+Par exemple, sous Windows pour activer l'enregistrement avec l'option `-R` : cliquez-droit sur le raccourci Jamulus et choisissez `Propriétés` > `Cible`. Ajoutez les arguments nécessaires au fichier `Jamulus.exe` :
+
+```shell
+"C:\Program Files (x86)\Jamulus\Jamulus.exe" -s -R "C:\chemin\vers\vos\enregistrements de bœufs"
+```
+
+Pour Macintosh, lancez une fenêtre de terminal et exécutez Jamulus avec les options souhaitées comme ceci:
```
Comment (gegeweb) [inline wiki/fr/fr-Command-Line-Options.md:18]: I know that Macintosh is used in the EN version, but it is really a wrong formulation.
But we can wait EN update.

**Review by gegeweb — APPROVED** [review]
Sounds good for me.

« Macintosh » sounds bad to me. But we can wait the fix on the EN version.

Thanks.

**Comment by gegeweb** [comment]
Ready to be merged for me too.

**Comment by trebmuh** [comment]
Thanks for the review.

**Comment by gegeweb** [comment]
> Thanks for the review.

And thank you for the correction.
Sorry again for botching this one!

**Pull request merged**